docs/reference/troubleshooting.asciidoc

```diff
@@ -56,6 +77,8 @@ include::troubleshooting/data/start-ilm.asciidoc[]
 
 include::troubleshooting/data/start-slm.asciidoc[]
 
+include::troubleshooting/data/restore-from-snapshot.asciidoc[]
```

Comment:
@Leaf-Lin This page does not exist. Is this a new page you want to add, or did you mean to link to https://www.elastic.co/guide/en/elasticsearch/reference/master/snapshots-restore-snapshot.html ?

Reply:
This refers to https://www.elastic.co/guide/en/elasticsearch/reference/master/restore-from-snapshot.html just published via https://github.com/elastic/elasticsearch/pull/87692